contrib/mesos/pkg/executor/executor.go

```diff
@@ -47,7 +47,6 @@ import (
 
 const (
 	containerPollTime = 1 * time.Second
```

Comment:
i wouldn't bother, this is probably going to be factored out at some point when we re-implement container monitoring